keadm/cmd/keadm/app/cmd/helm/installer.go

```diff
@@ -365,6 +365,9 @@ func (cu *KubeCloudHelmInstTool) checkProfile(baseHelmRoot string) error {
 func (cu *KubeCloudHelmInstTool) handleProfile(profileValue string) error {
 	// the current version
 	currentVersion := cu.ToolVersion.String()
+	if !strings.HasPrefix(currentVersion, "v") {
+		currentVersion = "v" + currentVersion
+	}
```

Comment:
Now the `--kubeedge-version=v1.12.0` flag has a higher priority than `--profile version=v1.11.0`?
In other words, which version we will use if we use both `--kubeedge-version=v1.12.0` and `--profile version=v1.11.0`? 1.12 or 1.11? @James-JLC  :)

Reply:
As I mentioned int #4388, we should not provide the option for uses to set kubeedge version using --profile flag, because we already have --kubeedge-version. And I don't know the historical reason --profile version=v1.xx.x flag is provided as an option, so I didn't make the changes here. But I do think it should be removed. As you said some optimizations need to be done right now or later.

Reply:
> As I mentioned in #4388, we should not provide the option for users to set kubeedge version using --profile flag, because we already have --kubeedge-version. And I don't know the historical reason --profile version=v1.xx.x flag is provided as an option, so I didn't make the changes here. But I do think it should be removed. As you said some optimizations need to be done right now or later.

there is not '--profile version' patamter show in kubeedge.io document . developer will prefer ' --kubeedge-version'

Reply:
The option is displayed in both [kubeedge.io document](https://kubeedge.io/en/docs/setup/keadm/#keadm-beta-init) and the output of command `keadm init`.

Reply:
/lgtm

Reply:
> profile version'

emmmm , my mistake . 
"you can make your custom values file here, and add flags like --profile version=v1.9.0 --set key=value to use this profile."